File: Modul 2/PRAK204/PRAK204-2310817310011-MuhammadRizky.py
phi = 22/7.0

inputUser = input("1 untuk input horizontal, 2 untuk input vertikal : \n")

if (inputUser == "1"):
    radiusBejana, heightBejana = input().split()
    radiusBejana, heightBejana = map(float, (radiusBejana, heightBejana))

    volumeBejana = phi * (radiusBejana * radiusBejana) * heightBejana

    luasPermukaanBejana = 2 * phi * radiusBejana * (radiusBejana + heightBejana)

    kelilingBejana = 2 * phi * radiusBejana
    print("Volume = %.2f"% volumeBejana)
    print("Luas = %.2f"% luasPermukaanBejana)
    print("Keliling = %.2f"% kelilingBejana)

elif (inputUser == "2"):
    radiusBejana = input()
    heightBejana = input()
    radiusBejana, heightBejana = map(float, (radiusBejana, heightBejana))

    volumeBejana = phi * (radiusBejana * radiusBejana) * heightBejana

    luasPermukaanBejana = 2 * phi * radiusBejana * (radiusBejana + heightBejana)

    kelilingBejana = 2 * phi * radiusBejana

    print("Volume = %.2f"% volumeBejana)
    print("Luas = %.2f"% luasPermukaanBejana)
    print("Keliling = %.2f"% kelilingBejana)
else:
    print("error")

File: Modul 2/PRAK204/PRAK204-2310817310011-MuhammadRizky.c
#include <stdio.h>
#include <math.h>

int main() {
    float radiusBejana, heightBejana, phi;
    phi = 22/7.0; //diblkng 7 hrs ditambah .0 
    scanf("%f %f", &radiusBejana, &heightBejana);

    float volumeBejana = phi * (radiusBejana * radiusBejana) * heightBejana;

    float luasPermukaanBejana = 2 * phi * radiusBejana * (radiusBejana + heightBejana);

    float kelilingBejana = 2 * phi * radiusBejana;

    printf("Volume = %.2f\n", volumeBejana);
    printf("Luas = %.2f\n", luasPermukaanBejana);
    printf("Keliling = %.2f", kelilingBejana);
    return 0;
}
